_data/docs.yml

```diff
@@ -167,6 +167,18 @@ apis:
       legacy: 1
       stable: 1
       nightly: 1
+  rapidspdf:
+    name: RAPIDSMPF
+    path: rapidsmpf
+    desc: 'RAPIDS Multi-Process Foundation (rapidsmp) a collection of multi-gpu, distributed memory algorithms written in C++ and exposed to Python'
+    ghlink: https://github.com/rapidsai/rapidsmpf
+    cllink: https://github.com/rapidsai/rapidsmpf/blob/main/CHANGELOG.md
```

Comment:
This link 404's. There is no `CHANGELOG.md` on branches `main`, `branch-25.06`, or `branch-25.08`.

Reply:
removed cllink and will add it back when we have a changelog -- probably after 25.06